### PR TITLE
docs(pt-br): translated git hooks page

### PR DESCRIPTION
**Pull request by CookieDasora**
## Summary

This PR adds the Portuguese translation of the Git Hooks guide as requested in #1421, adds the new homepage to portuguese version of the website and translates the guide Integrate in VCS.

## Test Plan

- Check if there is no missing translation in the translated files.
- Check if the translations are correct and if they adhere to the grammatical rules of Portuguese.

### CODE REVIEW

File: website/src/content/docs/pt-br/guides/integrate-in-vcs.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+```json title="biome.json"
+{
+  "vcs": {
+    "enabled": true,
+    "clientKind": "git"
+  }
+}
+```
+
+Essa configuração não faz **qualquer coisa em si**. Você precisa adicionar novos recursos.
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/guides/integrate-in-vcs.mdx:21]:
```suggestion
Essa configuração não faz **nada por si só**. Você precisa adicionar novos recursos.
```

What do you think?

File: website/src/content/docs/pt-br/guides/integrate-in-vcs.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+### Ignorando arquivos
+
+Isso é um recurso que permite que o Biome leia os arquivos ignorados do VCS e ignora todos os arquivos e pastas especificados nele. Esse é um recurso opcional, e você precisa ativar o campo `vcs.useIgnoreFile`:
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/guides/integrate-in-vcs.mdx:25]:
```suggestion
Esse é um recurso que permite que o Biome leia os arquivos ignorados do VCS e ignora todos os arquivos e pastas especificados nele. Esse é um recurso opcional, e você precisa ativar o campo `vcs.useIgnoreFile`:
```
To keep same as "Processando apenas arquivos modificados"

**Review by victor-teles — CHANGES_REQUESTED** [review]

File: website/src/content/docs/pt-br/guides/integrate-in-vcs.mdx (hunk 1 above)
Comment (CookieDasora) [reply website/src/content/docs/pt-br/guides/integrate-in-vcs.mdx:21]: Sounds good!

**Review by victor-teles — APPROVED** [review]
Looks good to me

**Pull request merged**